### PR TITLE
GP11 : ECDSA NIST test vectors

### PR DESCRIPTION
**Pull request by cedric-chaumont-st-dev**
Signed-off-by: Cedric Chaumont <cedric.chaumont@st.com>

### CODE REVIEW

File: host/xtest/xtest_4000.c [hunk 1]
```diff
@@ -3160,26 +3161,317 @@ static const struct xtest_ac_case xtest_ac_cases[] = {
...
+			    nist_186_2_ecdsa_testvector_73),
+	XTEST_AC_ECDSA_CASE(TEE_ALG_ECDSA_P521, TEE_MODE_VERIFY,
+			    nist_186_2_ecdsa_testvector_74),
+	XTEST_AC_ECDSA_CASE(TEE_ALG_ECDSA_P521, TEE_MODE_SIGN,
+			    nist_186_2_ecdsa_testvector_74),
+	XTEST_AC_ECDSA_CASE(TEE_ALG_ECDSA_P521, TEE_MODE_VERIFY,
+			    nist_186_2_ecdsa_testvector_75),
+	XTEST_AC_ECDSA_CASE(TEE_ALG_ECDSA_P521, TEE_MODE_SIGN,
+			    nist_186_2_ecdsa_testvector_75),
+	/* [K-163] - GP NOT SUPPORTED */
```
Comment (pascal-brand-st-dev) [inline host/xtest/xtest_4000.c]: Is it `[K-163]` or `[P-163]`?

**Comment by pascal-brand-st-dev** [comment]
Few questions and remarks:
* How long does it take to run all the tests? Can/Should we have an option to skip some of them?
* Is there a script to generate the .h file? If so, can we deliver it?
* xtest_4000_data.h to be cleaned

**Comment by cedric-chaumont-st-dev** [comment]
Update

**Comment by cedric-chaumont-st-dev** [comment]
Squashed. Verified.
+-----------------------------------------------------
34536 subtests of which 1 failed
687 test cases of which 1 failed
0 test case was skipped
TEE test application done!

Note: GP failing test 7671 (patch on going)

File: host/xtest/xtest_4000.c (hunk 1 above)
Comment (pascal-brand-st-dev) [inline host/xtest/xtest_4000.c]: Question is still there.

Comment (cedric-chaumont-st-dev) [inline host/xtest/xtest_4000.c]: K-163

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by cedric-chaumont-st-dev** [comment]
Rebase. Update git comment.

**Pull request merged**